Sources/JWTKit/Claims/GoogleHdClaim.swift

```diff
@@ -0,0 +1,15 @@
+public struct GoogleHdClaim: JWTClaim, Equatable, ExpressibleByStringLiteral {
```

Comment:
Is `Hd` short for anything? If so, using the full spelling here would be nice. Otherwise it should be `HD` if it's an abbreviation.

Reply:
It represents the G Suite domain of the user.  No idea why Google called it that.

Reply:
To the same vein as the previous comment...I think it's best to refer to it the same way that the docs do since it's not a simple snake to camel case conversion.